Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -69,22 +69,19 @@
   icon: /Textures/Interface/Alerts/Weightless/weightless.png
   name: Weightless
   description: >-
-    Gravity has ceased affecting you, and you're floating around aimlessly. You'll need something large and heavy, like a
-    wall or lattice, to push yourself off if you want to move. A jetpack would enable free range of motion. A pair of
-    magboots would let you walk around normally on the floor. Barring those, you can throw things, use a fire extinguisher,
-    or shoot a gun to move around via Newton's 3rd Law of Motion.
+    Gravity has ceased affecting you, and you're floating around aimlessly. Find something sturdy to hold onto, or expand force in a direction opposite of you. Magnetic boots or jetpacks will help you move with more control
```

Comment:
"or expand force in a direction opposite from you." -> Not good. Needs to be rewritten.

"Magnetic boots or jetpacks will" -> "Mag-boots or a jetpack would"